docker/mysql/Dockerfile

```diff
@@ -1,5 +1,37 @@
 FROM mysql
+
 # All scripts in docker-entrypoint-initdb.d/ are automatically
 # executed during container startup
 COPY docker/mysql/initdb.sql /docker-entrypoint-initdb.d/
-COPY db/rdbms/schema/latest/create_contest_db.sql /
+COPY db/rdbms/schema/v0/create_contest_db.sql /
```

Comment:
Out of curiosity:

Why do we call it `v0`. Will there be any other version except for `0`? And why isn't it just the first migration (why do we create the initial tables in a dedicated flow)?

Reply:
> Will there be any other version except for 0?

Every migration determines a new version. I named v0 the baseline. 

> why do we create the initial tables in a dedicated flow?

It just simplifies the transition to the migration scheme, where everybody who is working with ConTest is already known to be working on the same baseline. I can make the actual v0 a migration itself. Is going to be more painful to allow people to start using the migrations, but if you think v0 is too confusing I'll just do it.

Reply:
No-no. I was just wondering about the chain of thought. Thank you for the clarifications.